algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -20,49 +20,156 @@ import scala.collection.immutable.BitSet
 import scala.collection.JavaConverters._
 
 import com.googlecode.javaewah.{ EWAHCompressedBitmap => CBitSet }
+import com.googlecode.javaewah.IntIterator
 
 object RichCBitSet {
-  def apply(x: Int*) = {
-    CBitSet.bitmapOf(x.sorted: _*)
+  def apply(xs: Int*): CBitSet = fromArray(xs.toArray)
```

Comment:
I don't really know the details, but `Int*` has type `_*[Int]` or something, which is a seq, but not an Array as far as I know.